[Docs] Fix content of Hive security configuration page (#21175)
prestodb/presto: presto-docs/src/main/sphinx/connector/hive-security.rst

:ref:`Additional Information About Keytab Files.<hive-security-additional-keytab>`

HDFS wire encryption
--------------------

In a Kerberized Hadoop cluster with enabled HDFS wire encryption you can enable

> Presto to access HDFS by using below property.

Reviewer: ```suggestion
Presto to access HDFS by using the ``hive.hdfs.wire-encryption.enabled`` property.
```
Below and above can be ambiguous. Suggest specifying the exact name so the sentence has meaning by itself, when possible.

You should ensure that the keytab files have the correct permissions on every
node after distributing them.

Authorization
=============

> You can enable authorization checks for the :doc:`/connector/hive` by setting

Reviewer: ```suggestion
Enable authorization checks for the :doc:`/connector/hive` by setting
```
Imperative is simpler to read and understand, and clearer to the reader what action they must perform to achieve the goal.

                                                   In this mode, Presto enforces the authorization checks for
                                                   queries based on the privileges defined in Hive metastore.
                                                   To alter these privileges, use the :doc:`/sql/grant` and
                                                   :doc:`/sql/revoke` commands.
                                                   See :ref:`hive-sql-standard-based-authorization` for details.

``ranger``                                         Users are permitted to perform the operations as per the
                                                   authorization policies configured in Ranger Hive service.
                                                   See :ref:`hive-ranger-based-authorization` for details.
================================================== ============================================================

.. _hive-sql-standard-based-authorization:

SQL Standard Based Authorization
--------------------------------

When ``sql-standard`` security is enabled, Presto enforces the same SQL
standard based authorization as Hive does.

> Since Presto's ``ROLE`` syntax support matches the SQL standard, and

Reviewer: ```suggestion
Because Presto's ``ROLE`` syntax support matches the SQL standard and
```
Minor suggestion here: Since has meaning of both time and cause, Because means only cause.
